cookbooks/swift/recipes/setup.rb

```diff
@@ -86,7 +86,7 @@
 bash 'install pip' do
   code <<-EOF
     set -o pipefail
-    curl https://bootstrap.pypa.io/get-pip.py | python
+    curl https://bootstrap.pypa.io/get-pip.py | python#{node['use_python3'] && '3' || '2'}
```

Comment:
so this just changes the system pip command to install libs for py3 and everything flows from there... awesome

Reply:
Yup -- get pip on py3, then all the `pip install X` commands install on py3 and it all Just Works™ as best I can tell.